doc/_scripts/software_maturity/software_maturity_features.yaml

```diff
@@ -132,6 +132,12 @@ features:
         - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
         - BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001
+  fast_pair:
+    Initial Pairing: BT_FAST_PAIR
+    Subsequent pairing: BT_FAST_PAIR && EXPERIMENTAL
+    Battery Notification extension: BT_FAST_PAIR && EXPERIMENTAL
+    Personalized Name extension: BT_FAST_PAIR && BT_FAST_PAIR_PN
+    Find My Device Network extension: BT_FAST_PAIR && BT_FAST_PAIR_FMDN
```

Comment:
I wonder if we need to explicitly list `BT_FAST_PAIR` if the second listed option depends on it in Kconfig

Reply:
yes, maybe we don't have to do it

Reply:
I will improve it